Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -1332,12 +1332,15 @@ public SubscriptionDataConfig GetSubscription(Symbol symbol, TickType? tickType)
             SubscriptionDataConfig subscription;
             try
             {
+                // deterministic ordering is required here
+                var subscriptions = SubscriptionManager.Subscriptions.OrderBy(x => x.TickType);
```

Comment:
I wonder if it would be more performant to filter first by (x => x.Symbol == symbol). I image it will probably depend on each case.